ndmg/register.py

```diff
@@ -265,6 +267,9 @@ def gen_tissue(self):
         self.wm_mask = self.maps["wm_prob"]
         self.gm_mask = self.maps["gm_prob"]
         self.csf_mask = self.maps["csf_prob"]
+        reg_mri_pngs(self.csf_mask, self.gm_mask, self.wm_mask, self.namer)
```

Comment:
add a comment here to explain why this line is here

Reply:
you did not do this, please don't mark things as resolved if they aren't resolved.